lib/museum.rb

```diff
@@ -0,0 +1,20 @@
+class Museum
+  attr_reader :name, :exhibits
+
+  def initialize(name)
+    @name = name
+    @exhibits = []
+  end
+
+  def add_exhibit(exhibit)
+    @exhibits << exhibits
```

Comment:
```suggestion
    @exhibits << exhibit
```

This typo causes my tests to hang